python/tank/deploy/tank_commands/update.py

```diff
@@ -149,80 +217,92 @@ def run_interactive(self, log, args):
 # helper methods for update
     
     
-def check_for_updates(log, tk, env_name=None, engine_instance_name=None, app_instance_name=None, suppress_prompts=False):
+def check_for_updates(log, 
+                      tk, 
+                      env_name, 
+                      engine_instance_name, 
+                      app_instance_name, 
+                      external=None,
+                      preserve_yaml=False,
+                      suppress_prompts=False):
     """
     Runs the update checker.
+    
+    :param log: Python logger
+    :param tk: Toolkit instance
+    :param env_name: Environment name to update
+    :param engine_instance_name: Engine instance name to update
+    :param app_instance_name: App instance name to update
+    :param suppress_prompts: If True, run without prompting
+    :param preserve_yaml: Indicates that a comment preserving yaml parser 
```

Comment:
I would have preferred this parameter to be documented like suppress_prompts:

`If True, a comment preserving yaml parser is used.`